src/main/java/com/darzalgames/darzalcommon/data/ListUtility.java

```diff
@@ -0,0 +1,53 @@
+package com.darzalgames.darzalcommon.data;
+
+import java.util.List;
+
+import com.darzalgames.darzalcommon.math.SimpleMath;
+
+public class ListUtility {
+	
+	/**
+	 * Given a list of values, get the index of the element "paired with" the element at the provided index.
+	 * e.g. the elements at 0 and 1 are paired, 2 and 3 are paired, etc.  In an odd-sized list, the middle-most value is paired with itself.
+	 * @param values
+	 * @param index
+	 * @return
+	 */
+	public static <E> int getClosestPairIndex(List<E> values, int index) {
```

Comment:
This function (and it's friends) seem a bit too specific to 6SS to be library functions.